onnxruntime/core/providers/cuda/cuda_kernel.h

```diff
@@ -170,6 +170,10 @@ class CudaKernel : public OpKernel {
     return provider_->PerThreadDefaultCudnnHandle();
   }
 
+  inline cudaStream_t DefaultCudaStream() const {
+    return provider_->ComputeStream();
```

Comment:
Nit: It would be nice if we added a comment here for future readers documentating that this returns the CUDA EP level stream which may be different than the stream used to actually perform the compute task(s) of the CUDA kernel (That is supplied via the `OpKernelContext`). Probably a similar comment for the CUDA EP's `ComputeStream()` will go a long way as well.